### PR TITLE
Removed line break from code snippet that caused code to break in Cloud Shell

### PR DESCRIPTION
**Pull request by tm-g**
Not sure if this breaks character-per-line style requirements though

### CODE REVIEW

File: README-QWIKLABS.md [hunk 1]
```diff
@@ -191,8 +191,7 @@ Once the app has been deployed, it can be viewed in the [Workload](https://conso
 Incidentally, the endpoint can be programmatically acquired using the following command:
 
 ```console
-echo http://$(kubectl get svc tracing-demo -n default \
-  -ojsonpath='{.status.loadBalancer.ingress[0].ip}')
+echo http://$(kubectl get svc tracing-demo -n default -ojsonpath='{.status.loadBalancer.ingress[0].ip}')
```
Comment (bgeesaman) [inline README-QWIKLABS.md]: Recommend wrapping in quotes like: `echo "http://$(kubectl get svc tracing-demo -n default -ojsonpath='{.status.loadBalancer.ingress[0].ip}')"`

**Review by bgeesaman — COMMENTED** [review]
Minor nit

File: README-QWIKLABS.md (hunk 1 above)
Comment (tm-g) [reply README-QWIKLABS.md]: Thanks for the feedback! I've committed the change you recommended.

**Review by bgeesaman — APPROVED** [review]
/lgtm
/approve

**Pull request merged**